src/index.js

```diff
@@ -1,13 +1,25 @@
-import _ from "lodash";
-import "./style.css";
+import './style.css';
 
-function component() {
-  const element = document.createElement("div");
-  const btn = document.createElement("button");
+const todoList = document.querySelector('.todo-list');
+const todos = [
+  {
+    index: 1,
+    completed: true,
+    task: 'Wash the dish',
+  },
+  {
+    index: 2,
+    completed: true,
+    task: 'Go to the gym',
+  },
+];
 
-  element.innerHTML = _.join(["Hello", "webpack"], " ");
+todos.forEach((todos) => {
+  const li = document.createElement('li');
+  li.className = 'list-item';
 
-  return element;
-}
-
-document.body.appendChild(component());
+  li.innerHTML = `<input type="checkbox" class="check" /> ${todos.task} <span class="material-icons" id="close">
+close
+</span>`;
+  todoList.appendChild(li);
+});
```

Comment:
- [ ] Based on the project requirement, you are supposed to create a function that iterates over the tasks array and populates the HTML list item element for each task. See the example below 👇🏾 

```javascript
// ⚠️ Please note that students don't have to implement this function exactly like this, this is only "one" example of how it can be implemented.


const populateTasks = (task, parentElement) => {
  const { description, completed, index: id } = task;
  const innerHTML = `
         <p>${description}</p>
         <p>${completed ? 'completed' : 'not completed'}</p>`;
  const newTodoListElement = document.createElement('li');
  Object.assign(newTodoListElement, { innerHTML, id });
  parentElement.appendChild(newTodoListElement);
};
```

This is just an example, it is not compulsory to use the exact code above.